src/saurav/Test1_NonRepetingCharFromString.java

```diff
@@ -0,0 +1,32 @@
+package saurav;
+
+public class Test1_NonRepetingCharFromString {
+
+	int getCountOfChar(String input, char ch) {
+		int count = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (input.charAt(index) == ch) {
+				count++;
+			}
+		}
+		return count;
+	}
+
+	void firstNonRepetingChar(String input) {
+		char letter = ' ';
+		for (int index = 0; index < input.length(); index++) {
+			char ch = input.charAt(index);
+			int count = getCountOfChar(input, ch);
+			if (count == 1) {
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop